server/README.md

```diff
@@ -494,3 +511,28 @@ curl --location --request GET 'http://localhost:3333/api/v1/user'
     ]
 }
 ```
+
+## Get ws status (returns the status of ws connection mapped to the userID)
+
+### Example cURL Command:
+
+```
+curl --location --request GET 'http://localhost:3333/api/v1/status'
```

Comment:
Does it make sense to show an example where you provide the userId value in the header? And/or make it clear the default user will be used (like for all other commands)?